pkg/rate/rate.go

```diff
@@ -46,7 +46,7 @@ func (r *Rate) Add(count int) {
 // the rate is computed by first calculating how far along the current time is in the current window
 // as a ratio between 0 and 1.0. Then, the sum of currentWindowCount + (1 - ratio) * previousWindowCounter
 // is returned as the interpolated counter between the two windows.
-func (r *Rate) Rate() float64 {
+func (r *Rate) SyncRate() float64 {
```

Comment:
Why rename this? This rate pkg is not specific to chunks per seconds or sync rate.

Reply:
This was accidentally included by the rename action when I did renaming of the `depthmonitor.SyncReporter`. Fixed!

Reply:
🙏